src/CppUTest/JUnitTestOutput.cpp

```diff
@@ -135,6 +136,9 @@ void JUnitTestOutput::printCurrentTestStarted(const UtestShell& test)
         impl_->results_.tail_ = impl_->results_.tail_->next_;
     }
     impl_->results_.tail_->name_ = test.getName();
+    if (*(test.getProgressIndicator()) == '!') {
```

Comment:
Probably, I was at the time just trying to make it work.  Had thoughts of adding a status to Utest that would indicate not run, passed, failed or skipped, but never tried to do anything with it.